app/models/VatRateAndSales.scala

```diff
@@ -27,7 +27,14 @@ case class VatRateAndSales(
                     validFrom: LocalDate,
                     validUntil: Option[LocalDate] = None,
                     sales: Option[SalesAtVatRate]
-                  )
+                  ) {
+
+  lazy val rateForDisplay: String = if(rate.isWhole) {
+    rate.toString.split('.').headOption.getOrElse(rate.toString) + "%"
```

Comment:
Where is this used? Cant seem to find any usages. If its not can we remove? If it is, why wouldn't we show the full rate?

Reply:
ah sorry, I didnt spot the check if its whole!